pkg/gpu/probe.go

```diff
@@ -246,7 +246,7 @@ func (p *Probe) setupManager(buf io.ReaderAt, opts manager.Options) error {
 			{Name: cudaAllocCacheMap},
 			{Name: cudaSyncCacheMap},
 			{Name: cudaSetDeviceCacheMap},
-		}})
+		}}, "gpu", &ebpftelemetry.ErrorsTelemetryModifier{})
```

Comment:
remove the ErrorsTelemetryModifier. I will add it in a separate PR, after changing the ebpf side to use the wrappers

Reply:
This came in the merge, it was added in another PR: https://github.com/DataDog/datadog-agent/pull/30047